pyproject.toml

```diff
@@ -10,7 +10,7 @@ authors = [
     { name = "Dominique Beaini", email = "dominique@valencediscovery.com" },
 ]
 readme = "README.md"
-requires-python = ">=3.10"
+requires-python = ">=3.8"
```

Comment:
Change conda version as well??

Reply:
Done

Reply:
the env.yml file is only used in that repo. you can ofc put >=3.8 (I just used >=3.10 because i saw it somewhere in the old goli so i thought it was the minimum python version used).

@DomInvivo you can also ask the CI to test for multiple python versions in a matrix CI at https://github.com/datamol-io/graphium/blob/main/.github/workflows/test.yml#L19.

Reply:
also add it at https://github.com/datamol-io/graphium/blob/main/pyproject.toml#L25-L28